spec/features/categories_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
+require 'support/sign_in_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.